Documentation/operations/upgrade.rst

```diff
@@ -40,8 +40,7 @@ file.
       helm template |CHART_RELEASE| \\
         --namespace=kube-system \\
         --set preflight.enabled=true \\
-        --set agent.enabled=false \\
-        --set config.enabled=false \\
+        --set enabled=false \\
```

Comment:
I'm inclined to agree with @tgraf here as well, `--set enabled=false` doesn't imply anything about the agent, seems not very obvious what it means here. (I didn't inspect how this field is used across the templates though)

Reply:
I'm not sure what the right implementation should be. I am not a big fan of creating a top-level `agent` map with just a single key-value pair. Should all the agent settings scope there?

Reply:
Even `--set agent=false` is good

Reply:
your suggestion stands, --set agent=false